### PR TITLE
Fixes #17106: packer configuration doesn't work on debian10

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/17106

### CODE REVIEW

File: packer/files/.vimrc [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+" Format de la barre d'état (tronquée au début, fichier, flags,  :
+set statusline=%<%f%m\ %r\ %h\ %w%=%l,%c\ %p%%
+
+" indentation cfengine
+filetype plugin on
+":helptags ~/.vim/cf3_doc/
+au BufRead,BufNewFile *.cf set ft=cf3
+au BufRead,BufNewFile *.cf.2 set ft=cf3
+au BufRead,BufNewFile *.st set ft=cf3
+au BufRead,BufNewFile *.st.2 set ft=cf3
```
Comment (Fdall) [inline packer/files/.vimrc:129]: Could you add the following:
  au BufRead,BufNewFile *.ps1 set ft=ps1
  au BufRead,BufNewFile *.ps1.st set ft=ps1

File: packer/scripts/start_deb.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export DEBIAN_FRONTEND=noninteractive
+
+# package that should exist everywhere
+apt-get -y install zsh vim less curl binutils rsync tree ntp htop dos2unix zip python tree htop ldapscripts lsb-release apt-transport-https dos2unix
+apt-get -y install git || apt-get -y install git-core
+
+if hash service 2>/dev/null
+then
+  :
```
Comment (Fdall) [inline packer/scripts/start_deb.sh:13]: ?

File: packer/scripts/vagrant.sh [hunk 3]
```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
 
+set -xe
+
 # vagrant connection
 mkdir -p /home/vagrant/.ssh
-wget --no-check-certificate \
-    'https://raw.githubusercontent.com/hashicorp/vagrant/master/keys/vagrant.pub' \
+wget 'https://raw.githubusercontent.com/hashicorp/vagrant/master/keys/vagrant.pub' \
```
Comment (Fdall) [inline packer/scripts/vagrant.sh:7]: this will fail on very old debian/ubuntu

File: packer/common/template.dhall [hunk 4]
```diff
@@ -3,13 +3,17 @@
...
 
+
 -- Global Vars
 let common_scripts =
      [
-     "scripts/vagrant.sh"
-     ,"scripts/guestAdd.sh"
-     ,"scripts/cleanbox.sh"
-     ,"scripts/end.sh"
+     "scripts/start_deb.sh",
```
Comment (Fdall) [inline packer/common/template.dhall:10]: start_deb.sh and stop_deb.sh should not be in the common template but rather in a specific os base file, like the os one.

Comment (Fdall) [reply packer/common/template.dhall:10]: Something like the agent target (https://github.com/Normation/rudder-tests/blob/master/packer/agent/agent.dhall).
What do you think?

**Comment by peckpeck** [comment]
PR updated with a new commit

**Pull request merged**